python/python/ert_gui/simulation/models/iterated_ensemble_smoother.py

```diff
@@ -76,32 +76,35 @@ def runSimulations(self, arguments):
         analysis_iter_config = analysis_config.getAnalysisIterConfig()
         num_retries_per_iteration = analysis_iter_config.getNumRetries()
         num_tries = 0
-        current_iteration = 1
 
-        while current_iteration <= getNumberOfIterations() and num_tries < num_retries_per_iteration:
+        while True:
```

Comment:
Would not the old loop condition with `current_iteration <= getNumberOfIterations()` changed to `current_iteration < getNumberOfIterations()` yield same behaviour? If so, I think that would be much easier to read...

Reply:
I'll look into this; personally I kind of like the:
```
while True:

    if cond1: 
         break

    if cond2:
         break
```
pattern; but the is in the beholder. Anyway - will look into your suggestion.